__tests__/app.test.js

```diff
@@ -144,3 +160,64 @@ describe("PATCH: /api/articles/:article_id should update corresponding article w
       });
   });
 });
+
+describe("GET: /api/articles should return an array of article objects sorted by descending date and takes a topic query", () => {
+  test("200: should return an array of objects that are sorted by date in descending order by default ", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const articles = response.body;
+        expect(articles).toBeSortedBy("created_at", { descending: true });
+        articles.forEach((article) => {
```

Comment:
If you are using a forEach, you want to also test for the expected length of articles array to make sure that it is running for all the objects which would be more robust.